src/mds/CDir.cc

```diff
@@ -2922,8 +2935,6 @@ void CDir::scrub_initialize()
     }
   }
   scrub_infop->directory_scrubbing = true;
-
-  assert(scrub_local()); // TODO: handle failure
```

Comment:
Well, it's *very important* we have all the directory's contents in memory when we set up the scrub metadata, because otherwise we'll be skipping children. Obviously that should happen, but scrub_local() was a check to guarantee it. Given the low odds of otherwise detecting missing scrubs of descendants, that's a check I think we need to keep around unless there's a compelling reason not to.